.github/workflows/deploy-vitest-ui.yaml

```
@@ -0,0 +1,45 @@
+  pages: write
+  id-token: write
+
+concurrency:
+  group: pages
+  cancel-in-progress: false
+
+jobs:
+  deploy:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.ouputs.page_url }}
```

actionlint (1.7.4)

property "ouputs" is not defined in object type {conclusion: string; outcome: string; outputs: {page_url: string}}

(expression)